lib/Importer/ONNXModelLoader.cpp

```diff
@@ -1619,17 +1619,21 @@ Error ONNXModelLoader::loadConv1D(const ONNX_NAMESPACE::NodeProto &op,
 
   // Construct the Bias field.
   Constant *bias = nullptr;
-
+  NodeValue B;
   // Check if we have a serialized bias vector.
   if (op.input_size() > 2) {
     auto &biasTensorName = op.input(2);
-    // Load the serialized bias vector.
-    ASSIGN_VALUE_OR_RETURN_ERR(bias, getConstantByName(biasTensorName));
+    // Load the serialized bias vector as constant or NodeValue.
+    bias = getConstantByNameOrNull(biasTensorName);
+    if (!bias) {
+      ASSIGN_VALUE_OR_RETURN_ERR(B, getNodeValueByName(biasTensorName));
+    }
```

Comment:
Is it not sufficient to just use getNodeValueByName here to assign the NodeValue `B` and use everywhere, and get rid of `Constant *bias`? Constants should also be registered as NodeValues too I believe. I.e.
```suggestion
    ASSIGN_VALUE_OR_RETURN_ERR(B, getNodeValueByName(biasTensorName));
```